.teamcity/_self/projects/WPComTests.kt

```diff
@@ -212,102 +213,40 @@ fun gutenbergBuildType(screenSize: String, buildUuid: String): BuildType {
 	}
 }
 
-fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String ): BuildType {
-    return BuildType {
-		id("WPComTests_gutenberg_Playwright_$targetDevice")
-		uuid=buildUuid
-		name = "Playwright E2E Tests ($targetDevice)"
-		description = "Runs Gutenberg E2E tests as $targetDevice using Playwright"
-
-		artifactRules = """
-			logs.tgz => logs.tgz
-			screenshots => screenshots
-			trace => trace
-		""".trimIndent()
-
-		vcs {
-			root(Settings.WpCalypso)
-			cleanCheckout = true
-		}
-
-		params {
+fun gutenbergPlaywrightBuildType( targetDevice: String, buildUuid: String ): E2EBuildType {
+    return E2EBuildType (
+		buildId = "WPComTests_gutenberg_Playwright_$targetDevice",
+		buildUuid = buildUuid,
+		buildName = "Playwright E2E Tests ($targetDevice)",
+		buildDescription = "Runs Gutenberg e2e tests on $targetDevice size",
+		testGroup = "gutenberg",
+		buildParams = {
 			text(
-				name = "URL",
+				name = "env.URL",
```

Comment:
Does this need to use `"%env.URL%"`? I seem to remember having to use that to access params in the past.

Reply:
It's been a few weeks so it's a bit rusty, but what I recall is that TeamCity configuration params do not get propagated to the builds, so when we go to set the environment variable to %URL% it tells me that the value is undefined.